release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -579,7 +585,10 @@ In the table below, features are marked with the following statuses:
 [id="ocp-4-5-known-issues"]
 == Known issues
 
-
+* The image registry of a disconnected cluster cannot pull images from mirrors.
```

Comment:
The deploy pod can't pull mirror image by digest id whose imagestream enabled pull-through policy.